assets/prometheus-k8s/rules.yaml

```diff
@@ -92,6 +92,11 @@ spec:
     - expr: |
         sum(rate(container_cpu_usage_seconds_total{job="kubelet", metrics_path="/metrics/cadvisor", image!="", container!="POD"}[5m])) by (namespace)
       record: namespace:container_cpu_usage_seconds_total:sum_rate
+    - expr: |
+        sum by (cluster, namespace, pod, container) (
+          rate(kube_pod_container_status_last_terminated_reason{reason="OOMKilled",job="kube-state-metrics"})
```

Comment:
Why are you suddenly interested in the rate and not absolute value?

Reply:
refactored... I think I did it right...